server/routes/approvingLicences/handlers/approvalCases.test.ts

```diff
@@ -112,6 +117,17 @@ describe('Route Handlers - Approval - case list', () => {
         description: 'Birmingham (HMP)',
       },
     ] as PrisonDetail[])
+
+    licenceService.getLicence.mockResolvedValue({
+      id: 1,
+      typeCode: 'AP',
+      additionalLicenceConditions: [],
+      additionalPssConditions: [],
+      bespokeConditions: [],
+      isVariation: false,
+      conditionalReleaseDate: '2022-1-5',
+      actualReleaseDate: '2022-1-3',
```

Comment:
wouldn't the licence service return dates in `dd/MM/yyy` format?

Reply:
Looking at other examples, yes, but I can't seem to follow the code path that handles this? In the DB they're stored as yyyy-mm-dd, and I can't seem to see where the conversion happens on the 'get licence' side.